src/logcollector/logcollector.c

```diff
@@ -1514,3 +1515,19 @@ void w_create_input_threads(){
 #endif
     }
 }
+
+void key_lock_init()
```

Comment:
Use a proper name for this function, like `files_lock_init()`, please.

Reply:
Changed!